### PR TITLE
Vpn services info

### PR DESCRIPTION
**Pull request by irina-pereiaslavskaia**

### CODE REVIEW

File: plugins/modules/vpn_services_info.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    description: Name or ID of router.
+    type: str
+  status:
+    description: Specifies whether the VPN service is currently operational.
+    choices: [active, down, build, error, pending_create, pending_update, pending_delete]
+    type: str
+  subnet:
+    description: Name or ID of subnet.
+    type: str
+  tenant_id:
```
Comment (anton-sidelnikov) [inline plugins/modules/vpn_services_info.py:46]: call it project_id

Comment (anton-sidelnikov) [inline plugins/modules/vpn_services_info.py:46]: call it project_id

File: tests/integration/targets/vpn_services_info/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        router: "fake_router"
+      register: vpn_services_info
+      ignore_errors: yes
+
+    - name: assert result
+      assert:
+        that:
+          - vpn_services_info is not success
+          - vpn_services_info is not changed
+          - 'vpn_services_info.msg == "Router fake_router not found"'
```
Comment (anton-sidelnikov) [inline tests/integration/targets/vpn_services_info/tasks/main.yaml:54]: add test with multiple filers

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: plugins/modules/vpn_services_info.py (hunk 1 above)
Comment (irina-pereiaslavskaia) [reply plugins/modules/vpn_services_info.py]: Done

File: tests/integration/targets/vpn_services_info/tasks/main.yaml (hunk 2 above)
Comment (irina-pereiaslavskaia) [reply tests/integration/targets/vpn_services_info/tasks/main.yaml]: Done

File: tests/sanity/ignore-2.9.txt [hunk 3]
```diff
@@ -58,6 +59,7 @@ plugins/modules/lb_member.py validate-modules:missing-gplv3-license
 plugins/modules/lb_member_info.py validate-modules:missing-gplv3-license
 plugins/modules/lb_healthmonitor.py validate-modules:missing-gplv3-license
 plugins/modules/lb_healthmonitor_info.py validate-modules:missing-gplv3-license
+plugins/modules/vpn_services_info.py validate-modules:missing-gplv3-license
```
Comment (anton-sidelnikov) [inline tests/sanity/ignore-2.9.txt:62]: why duplicate?

**Review by tischrei — DISMISSED** [review]
please wait for a second approval.

File: plugins/modules/vpn_service_info.py [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    description: Specifies whether the VPN service is currently operational.
+    choices: [active, down, build, error, pending_create, pending_update, pending_delete]
+    type: str
+  subnet:
+    description: Name or ID of subnet.
+    type: str
+  project_id:
+    description: Specifies the project ID
+    type: str
+  vpn_service:
```
Comment (anton-sidelnikov) [inline plugins/modules/vpn_service_info.py:49]: check this parameter

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: tests/sanity/ignore-2.9.txt (hunk 3 above)
Comment (irina-pereiaslavskaia) [reply tests/sanity/ignore-2.9.txt]: Delete it

**Review by anton-sidelnikov — APPROVED** [review]

**Review by tischrei — APPROVED** [review]

**Comment by irina-pereiaslavskaia** [comment]
regate

**Comment by irina-pereiaslavskaia** [comment]
regate

**Pull request merged**